system-probe_arm64/Dockerfile

```diff
@@ -50,16 +47,16 @@ RUN apt-get update && apt-get install -y --no-install-recommends apt-utils && ap
         wget \
         xz-utils
 
-RUN wget -O /bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/v1.5.4/gimme
-RUN echo "03b295636d4e22870b6f6e9bc06a71d65311ae90d3d48cbc7071f82dd5837fbc  /bin/gimme" | sha256sum --check
-RUN chmod +x /bin/gimme
-RUN gimme $GIMME_GO_VERSION
+ENV GO_VERSION $GIMME_GO_VERSION
```

Comment:
Is `GO_VERSION` needed for someone else, or could you use `GIMME_GO_VERSION` in the download command below?

Reply:
And I wanted to keep the ENV=ARG pattern that was here before as well

Reply:
Should we replace `ARG GIMME_GO_VERSION` with `ARG GO_VERSION` then? You cannot really use a different version from the command line because the hashes won't match.

Reply:
Yeah that's a good point, let me do that